src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -313,7 +313,7 @@ private bool IsTextualTriggerCharacter(ICompletionService completionService, cha
             return completionService.IsTriggerCharacter(previousPosition.Snapshot.AsText(), previousPosition, GetCompletionProviders(), options);
         }
 
-        private bool IsCommitCharacter(char ch)
+        private bool IsCommitCharacter(char typedChar)
```

Comment:
Would a better approach to be to not even call this in the below case?  Here's what i'm thinking:

if the caret is at hte start of the filter span, should we even have anything selected strongly in the completion list?  And if we don't have something strongly selected, we shouldn't commit.

So the preferred behavior falls out not by saying "no this is not a commit character", but rather "we couldn't even commit here if we wanted to because there's no item selected.